apps/test-suite/app.json

```diff
@@ -8,6 +8,9 @@
     "platforms": ["android", "ios", "web"],
     "ios": {
       "bundleIdentifier": "io.expo.testsuite"
+    },
+    "android": {
+      "useNewNotificationsAPI": true
```

Comment:
Sorry for not finding this out earlier, I've looked at the code in `universe` and it looks like it's more common to write it like `Api` in camelcase

Reply:
Hm, maybe `nextNotificationsApi` is even better? I don't know, what do you think @bbarthec?